utility/CMakeLists.txt

```diff
@@ -1 +1,9 @@
-# Utils
\ No newline at end of file
+# Utils
+project(jsonWork)
```

Comment:
Utility folder is meant to be place for all utilites not only for json. Also name is not needed in all subdirectories.